public/css/widget/state-change.less

```diff
@@ -32,3 +32,12 @@
     }
   }
 }
+
+.state-change.detail-history {
+  .state-ball.ball-size-m {
```

Comment:
Please don't use a special class here. Combine it with the selector `.ball-size-l ~ .ball-size-ml` (`ml` is from https://github.com/Icinga/ipl-web/pull/45)